src/app/item-page/field-components/metadata-values/metadata-values.component.html

```diff
@@ -23,6 +23,14 @@
     </a>
 </ng-template>
 
+<!-- Render value as a link with icon -->
+<ng-template #linkImg let-img="img" let-value="value">
+  <a [href]="value" class="dont-break-out ds-simple-metadata-link" target="_blank">
+    <img alt="metadata logo" height="24" src="{{img}}"/>
```

Comment:
Ideally this should have a configurable `alt` value for accessibility.  So, for example, for ROR, this should say `alt="ROR logo"` or similar.

Alternatively, maybe this `alt` could be based on the field `label`?  So, maybe this could just be something like `alt="{{label | translate}} logo"`??

Finally, is it possible to remove the hardcoded `height="24"` and use CSS instead?  It'd be nice if sites can control the height of these logos **without having to edit the HTML**.